bin/getOrigins.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const axios = require('axios');
```

Comment:
I don't see the point of using `axios` when you can default `https` node module : 
```javascript
function getOrigins() {
  return new Promise((resolve, reject) => {
    https.get('', (res) => {
      let result = ''
      res.on('data', (data) => result += data)
      res.on('close', () => resolve(JSON.parse(result)))
      res.on('error', err => reject(err))
    })
  })
}
```

Reply:
Updated this.